README.md

```diff
@@ -45,10 +45,18 @@ Command line usage:
     wlc download
     wlc upload
 
-Configuration is stored in `~/.config/weblate`:
+Configuration is stored in `~/.config/weblate`. The key/values (`retries`,
+`timeout`, `method_whitelist`, `backoff_factor`, `status_forcelist`) are closely
+coupled with the `urllib3 parameters`\_ and allows the user to configure request
```

Comment:
It was supposed to be a link here?  The underscore itself definitely looks wrong.

Reply:
I think it's just a typo